src/status_im/native_module/impl/module.cljs

```diff
@@ -167,3 +168,8 @@
 (defn update-mailservers [enodes on-result]
   (when status
     (call-module #(.updateMailservers status enodes on-result))))
+
+(defn rooted-device? [callback]
+  (if (and platform/ios? status)
+    (call-module #(.isDeviceRooted callback))
```

Comment:
should it be `(call-module #(.isDeviceRooted status callback))` ?

Reply:
great point, also I see this logic is off a bit

Reply:
fixed it